roles/gui/node/tasks/install_dir_pkg.yml

```diff
@@ -0,0 +1,99 @@
+---
+# Dir package installation method
+
+- block:  ## run_once: true
+    - name: install | Stat local installation package
```

Comment:
Are we able to distinguish `install |` in this task file from others?  This is `install_dir_pkg.yml` .  I'm just making sure that if users open issues and we do a `git grep "Stat local installation package"` we get a uniq hit as to which task is being executed, instead of multiple hits and we're not 100% sure which path the playbook took.